docs/features/writing-different-telemetry-types.md

```diff
@@ -75,6 +76,44 @@ using (var measurement = DependencyMeasurement.Start())
 
 Note that we have an `LogServiceBusTopicDependency` to log dependency logs for an Azure Service Bus Topic and an `LogServiceBusDependency` to log Azure Service Bus logs where the entity type is not known.
 
+### Measuring Azure Table Storage Dependencies
+
+We allow you to measure Azure Table Storage dependencies.
+
+Here is how you can report an Azure Table Storage dependency:
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Collumn", "productID" }
+};
+
+var durationMeasurement = new Stopwatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+_logger.LogTableStorageDependency(tableName: "orders", isSuccessful: true, startTime, durationMeasurement.Elapsed, telemetryContext);
```

Comment:
This should also require the name of the storage account

Reply:
Ok, added that.